modules/heat_conduction/src/kernels/JouleHeatingSource.C

```diff
@@ -0,0 +1,53 @@
+#include "JouleHeatingSource.h"
+
+template<>
+InputParameters validParams<JouleHeatingSource>()
+{
+  InputParameters params = validParams<HeatSource>();
+  params.addCoupledVar("elec", "Electric potential for joule heating.");
+  params.addCoupledVar("args", "Vector of arguments of the diffusivity");
+  params.addParam<MaterialPropertyName>("electrical_conductivity", "electrical_conductivity", "material property providing electrical resistivity of the material.");
```

Comment:
You don't, I can. Maybe @permcody can add you to the collaborators group for which tests are run automatically.